js/build.js

```diff
@@ -1,6 +1,8 @@
 Fliplet.Widget.instance('login-ds', function(data) {
   var $container = $(this);
-
+  var $passwordInputs = $('.profile_password, .new-password, .confirm-password');
```

Comment:
This suggests that all the password visibilities on the page are being controlled in one go. There are 2 problems with this.

1. It should be specific to the widget instance, e.g. `$container.find(...)` so that each instance can have its separate behaviour.
2. https://weboo.atlassian.net/browse/OD-746?focusedCommentId=27859 suggests that we don't want them all to be toggled together.
3.